wsmaster/wsmaster-local/src/test/java/org/eclipse/che/api/local/LocalSnapshotDaoTest.java

```diff
@@ -80,7 +83,7 @@ private SnapshotImpl createSnapshot() {
         return SnapshotImpl.builder()
                            .generateId()
                            .setType("docker")
-                           .setInstanceKey(new DummyInstanceKey())
+                           .setMachineSource(machineSource)
```

Comment:
What is the purpose of mock serialization ?
This test is for checking whether objects correctly serialized/deserialised because we had problems with it.